docs/releases/1.10.txt

```diff
@@ -131,6 +131,9 @@ Minor features
   :attr:`MultiLineString.closed
   <django.contrib.gis.geos.MultiLineString.closed>` properties.
 
+* The :doc:`GeoJSON serializer </ref/contrib/gis/serializers>` now outputs by
+  default the primary key of objects in the ``properties`` dictionary.
```

Comment:
chop "by default" and add: "if specific fields aren't specified" to the end of the sentence?